### DISCUSSION TITLE
How to add License to the project ?

### DISCUSSION THREAD

**Discussion by h3cksamrat** [EddieHubCommunity/support#244] [Q&A] [answered]
I raised an [issue](#243 ) to review my [open-source project](https://github.com/h3cksamrat/FacebookAutomatedComment). There @BOLT04 advised me to add MIT License.
 
How can I add a License to the project?

**Comment by eddiejaoude** [comment, marked as answer]
If you go to `insights` tab, then `community`, you can add these types of files from there. Also GitHub has a good website on picking a license http://choosealicense.com 

<img width="1885" alt="Screenshot 2020-09-22 at 11 46 42" src="https://user-images.githubusercontent.com/624760/93873245-500ee180-fcc9-11ea-8fa8-05a209791c2d.png">

**Comment by h3cksamrat** [comment]
Thank You @eddiejaoude I added it to my project.

**Reply by eddiejaoude** [reply]
Great 👍

**Reply by eddiejaoude** [reply]
Btw @h3cksamrat you should add a GitHub profile picture it will make your account look more professions
